### ISSUE TITLE
Error 100 Internal Server Error (prod)

### ISSUE THREAD

**Issue by LauraErhard** [locdb/locdb-frend#426]
I wanted to save new coordinates and got the 500 Internal Server Error. 
![500 internal error](https://user-images.githubusercontent.com/32642653/48000051-b35aef00-e104-11e8-95e4-526d177534ca.png)

**Comment by LauraErhard** [comment]
Http failure response for https://locdb.bib.uni-mannheim.de/locdb/correctReferencePosition?scanId=5bbdfb48fb6737739386065c&coordinates=228%20660%201540%20755:

**Comment by kleinann** [comment]
Yes, the same here. It's not possible to save new bounding boxes.
[Display][error] Error while uploading new coordinates: Http failure response for https://locdb.bib.uni-mannheim.de/locdb/correctReferencePosition?scanId=5bbdfb48fb6737739386065d&coordinates=239%202113%201570%202248: 500 Internal Server Error

**Comment by LauraErhard** [comment]
I just wanted to note, that in the demo version the saving of new coordinates works. So maybe the issue is only in the production system?!

**Comment by kleinann** [comment]
I still get the error when trying to save bounding boxes in production:
[Display][error] Error while uploading new coordinates: Http failure response for https://locdb.bib.uni-mannheim.de/locdb/correctReferencePosition?scanId=5bbdfb48fb6737739386065d&coordinates=238%202109%201573%202253: 500 Internal Server Error

**Comment by anlausch** [comment]
In the cases you are presenting an error is returned from the reference extraction component. I tried it for other cases and it worked. Remember that the scan has to be processed by the reference extraction component first and has to be in its internal db. If the reference extraction component is set back in the meantime or when the scan was not processed yet, we cannot add any boxes.

**Comment by kleinann** [comment]
In all the cases that I tried, I worked in the Scan Inspector with a scan that had already some bounding boxes, so I guess that it must have been processed by the reference extraction component. I just tried again: first the boxes weren't saved, then the error happened againg. The scan looks like this:
![grafik](https://user-images.githubusercontent.com/33151732/48768213-e088cf00-ecb8-11e8-8d35-7ccd72897825.png)

**Comment by LauraErhard** [comment]
I just uploaded three new chapters (to prod) to test this again. Now I can't adjust the boxes at all. The curser doesn't even change to the two arrows. On top of that I can't move the boxes around. Deleting boxes seems to work.

**Comment by LauraErhard** [comment]
Strangely now it works, at least for a few examples. But now I got the 503 error again.

**Comment by LauraErhard** [comment]
Okay, after correcting a few boxes with no problem, I got the 500 error again and can't save new coordinates:
![500 coordinates](https://user-images.githubusercontent.com/32642653/48779211-6156c380-ecd7-11e8-9c84-caedcc14c8c6.png)
